Audio/DirectXTKAudio_Desktop_2015_DXSDK.vcxproj

```diff
@@ -48,25 +48,25 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v140</PlatformToolset>
```

Comment:
You updated this to 2019 instead of using the 2019 files. I don't have a 2019 DXSDK version, but it could be added easily enough.